docs/recipes/reasonml.md

```diff
@@ -0,0 +1,43 @@
+# [ReasonML](https://reasonml.github.io) / [BuckleScript](https://bucklescript.github.io)
+
+## Installation
+
+```sh
+npm install --save-dev ava
+```
+
+Then add `ava` to `bs-dev-dependencies` in your `bsconfig.json`:
+```js
+{
+  ...
+  "bs-dev-dependencies": ["ava"]
+}
+```
+
+## Getting started
+
+```ml
+open Sync;
+
+test("Should pass", t => {
```

Comment:
We don't tend to put "should" in test titles. "1 equals 1" would be more AVA-like.